packages/gitbook/src/components/hooks/useScrollActiveId.ts

```diff
@@ -8,9 +8,10 @@ export function useScrollActiveId(
     options: {
         rootMargin?: string;
         threshold?: number;
-    } = {}
+        active: boolean;
+    } = { active: true }
 ) {
-    const { rootMargin, threshold = 0.5 } = options;
+    const { rootMargin, threshold = 0.5, active } = options;
```

Comment:
Actually `active` is a bit confusing as a named compared to `enabled` as there is `activeId` in it :D but it's fine